lib/session.js

```diff
@@ -1,11 +1,21 @@
 const debug = require('debug')('telegraf:session-redis')
 const redis = require('redis')
 
+const getRedisSessionKey = (ctx) => {
```

Comment:
I beleive we good with default implementation, cause this change gonna break a lot of bots :)

Reply:
But we definitely need to merge provided typings!